scripts/format_disk_benchmark.pl

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/perl
+
+# Usage
+#
+# 1. Run benchmarking
+#
+#   $ cchq <env> run-shell-command -b <groups> 'wget -q -O /root/iolatency https://raw.githubusercontent.com/brendangregg/perf-tools/master/iolatency; chmod +x /root/iolatency; hostname; cd /opt/data/; iostat; dd bs=8192 if=/dev/zero of=/opt/data/testfile count=125000 conv=fdatasync; dd bs=8192 if=/opt/data/testfile of=/dev/null iflag=nocache count=125000; rm /opt/data/testfile; ls -l `mount | grep "on /opt/data type ext4" | cut -d\  -f 1 | cut -d/ -f 4 | cut -d- -f 1 | xargs -I "{}" bash -c "grep \"{}\" <(pvs)" | cut -d\  -f 3` | cut -d\  -f 5,6 | sed "s/ //" | xargs -I "{}" /root/iolatency -T 10 1 -d {}; rm /root/iolatency' > disk-all-benchmark
```

Comment:
we should change that `master` to a commit sha. maybe also bring it into the repo.

Reply:
:+1: I mostly just wanted to put this somewhere with the changes I made